paket-files/haf/YoLo/YoLo.fs

```diff
@@ -622,7 +622,7 @@ module App =
 
   /// Get the assembly resource
   let resource name =
-#if DNXCORE50
+#if NETSTANDARD1_5
```

Comment:
@theimowski yes, `dnxcore50` is deprecated (but work, and you can use `imports` ), `netstandard1.5` is the new fashion.
check the long [Documentation/architecture/net-platform-standard.md](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md#deprecated-monikers)
or just do `dotnet new` , that's always the recommended framework. 
the .NET CLI migrated to `netstandard , aspnet is doing it now ( the `dnxcore50` was their target framework, the base of .net cli and .netcore work)